project.clj

```diff
@@ -27,21 +28,14 @@
                  [puppetlabs/ssl-utils "0.8.1"]
                  [puppetlabs/dujour-version-check "0.1.2" :exclusions [org.clojure/tools.logging]]
                  [puppetlabs/http-client "0.4.4"]
-                 [org.jruby/jruby-core "1.7.20.1"
-                  :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
-                 ;; jffi and jnr-x86asm are explicit dependencies because,
-                 ;; in JRuby's poms, they are defined using version ranges,
-                 ;; and :pedantic? :abort won't tolerate this.
-                 [com.github.jnr/jffi "1.2.9"]
-                 [com.github.jnr/jffi "1.2.9" :classifier "native"]
-                 [com.github.jnr/jnr-x86asm "1.0.2"]
```

Comment:
Well, I left `:pedantic? :abort` in our project.clj and it didn't barf when I ran `lein` anything against it.  And it did appear that their poms pin specific versions for jffi...

https://github.com/jruby/jruby/blob/9.0.4.0/pom.xml#L116
https://github.com/jruby/jruby/blob/9.0.4.0/core/pom.xml#L166

... and jnr-x86asm ...

https://github.com/jruby/jruby/blob/9.0.4.0/core/pom.xml#L117

Figured it would be better to just pull in `jruby-core` and not mention those other dependencies since we don't have anything that calls into them directly AFAIK.